internal/common/quota.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/osbuild/image-builder/internal/db"
+)
+
+const (
+	day                  time.Duration = 24 * time.Hour
+	week                 time.Duration = 7 * day
+	defaultSlidingWindow time.Duration = 2 * week
+	defaultQuota         int           = 100
+)
+
+type quota struct {
+	Quota         int           `json:"quota"`
+	SlidingWindow time.Duration `json:"slidingWindow"`
+}
+
+// Returns true if the number of requests made by accountNumber during a sliding window is inferior to a threshold.
```

Comment:
"inferior to a threshold" sounds a bit unclear to me, how about "below a threshold".

Reply:
Yeah, this might be literal translation issues ^^